### ISSUE TITLE
add datadog trace search and analytics 

### ISSUE THREAD

**Issue by broderF** [census-instrumentation/opencensus-java#1835]
is it possible to use the opencensus-java api for datadog trace search and analytics?

**Comment by songy23** [comment]
Yes with the Datadog tracing exporter, please take a look at the readme: https://github.com/census-instrumentation/opencensus-java/tree/master/exporters/trace/datadog.

**Comment by broderF** [comment]
Yes tracing in general is working for me, but i am not able to enable the Trace Search and Analytics Feature (https://docs.datadoghq.com/tracing/trace_search_and_analytics/?tab=java)
